lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,148 @@
+# (c) 2017, Jason Vanderhoof <jason.vanderhoof@cyberark.com>, Oren Ben Meir <oren.benmeir@cyberark.com>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: conjur_variable
+    version_added: "2.5"
+    short_description: Fetch credentials from CyberArk Conjur.
+    description:
+      - Retrieves credentials from Conjur using the server's Conjur identity. More information about Conjur can be found at https://www.conjur.org/.
+    requirements:
+      - The machine running Ansible has been given a Conjur identity.
+    options:
+      _term:
+        description: Variable path
+        required: True
+"""
+
+EXAMPLES = """
+  - debug
+    msg: {{ lookup('conjur_variable', '/path/to/secret') }}
+"""
+
+RETURN = """
+  _raw:
+    description:
+      - Value stored in Conjur.
+"""
+
+import os.path
+from ansible.errors import AnsibleError
+from ansible.plugins.lookup import LookupBase
+from base64 import b64encode
+from netrc import netrc
+from os import environ
+from time import time
+from ansible.module_utils.six.moves.urllib.parse import quote_plus
+import yaml
+
+from ansible.module_utils.urls import open_url
+
+
+try:
+    from __main__ import display
+except ImportError:
+    from ansible.utils.display import Display
+    display = Display()
+
+
+# Load Conjur configuration from either `/etc/conjur.conf` or `~/.conjurrc`
+def _load_configuration():
+    for location in ['~/.conjurrc', '/etc/conjur.conf']:
```

Comment:
Note that a _plugin option_ should be used for this configuration path. This would require:
- add a new parameter in plugin metadata ([example](https://github.com/ansible/ansible/blob/03794864c2c3267f1fbff6ac49e2d883090284ad/lib/ansible/plugins/connection/ssh.py#L188))
- use `get_option` method in order to fetch option value ([example](https://github.com/ansible/ansible/blob/03794864c2c3267f1fbff6ac49e2d883090284ad/lib/ansible/plugins/connection/ssh.py#L973))